src/approve-prs.ts

```diff
@@ -22,6 +22,9 @@ async function processMethod(repository: GitHubRepository, pr: PullRequest) {
   try {
     await repository.approvePullRequest(pr);
   } catch (err) {
+    console.warn(
+      `    error trying to approve pull request ${pr.html_url}: ${err}`
```

Comment:
It won't, but this is what we do for all other commands, we just missed it here in this place.

I believe it will still show the main reason, e.g. 401 or anything like that.

Reply:
Perhaps compromise and just print `err.stack`?